src/main/java/org/mule/extension/file/internal/command/LocalReadCommand.java

```diff
@@ -71,12 +75,47 @@ public LocalReadCommand(LocalFileSystem fileSystem) {
     return read(config, fileAttributes, lock, timeBetweenSizeCheck);
   }
 
+  /**
+   * {@inheritDoc}
+   */
+  @Override
+  public Result<InputStream, LocalFileAttributes> read(FileConnectorConfig config, String filePath, Long timeBetweenSizeCheck,
+                                                       Long lockTimeout) {
+    Path path = resolveExistingPath(filePath);
```

Comment:
this code is repeated from the method above, try to move the repeated code into another method

Reply:
Fixed.